### PR TITLE
Fix lspci not installed by default in debian 9 backport

### PR DESCRIPTION
**Pull request by LiliDeng**
Found in weekly performance testing.
Run against Debian 9 - pass
Run against ubuntu - pass

### CODE REVIEW

File: Libraries/CommonFunctions.psm1 [hunk 1]
```diff
@@ -797,6 +797,15 @@ function Verify-MellanoxAdapter($vmData)
 	$mellanoxAdapterDetected = $false
 	while ( !$mellanoxAdapterDetected -and ($retryAttempts -lt $maxRetryAttemps))
 	{
+		$output = Run-LinuxCmd -ip $vmData.PublicIP -port $vmData.SSHPort -username $user -password $password -command "lspci" -runAsSudo -ignoreLinuxExitCode
+		if($output -imatch "command not found") {
+			Run-LinuxCmd -ip $vmData.PublicIP -port $vmData.SSHPort -username $user -password $password -command "if grep Debian /etc/issue;then apt-get install -y pciutils; fi" -runAsSudo -RunInBackGround
```
Comment (lizzha) [inline Libraries/CommonFunctions.psm1]: it can be general step to all distros to install pciutils package

**Comment by LiliDeng** [comment]
Testing is on the way

**Comment by LiliDeng** [comment]
image

File: Libraries/CommonFunctions.psm1 [hunk 2]
```diff
@@ -797,6 +797,10 @@ function Verify-MellanoxAdapter($vmData)
 	$mellanoxAdapterDetected = $false
 	while ( !$mellanoxAdapterDetected -and ($retryAttempts -lt $maxRetryAttemps))
 	{
+		Write-LogInfo "Install package pciutils to use lspci command."
+		.\Tools\dos2unix.exe -q .\Testscripts\Linux\utils.sh
+		Upload-RemoteFile -uploadTo $vmData.PublicIP -port $vmData.SSHPort -username $user -password $password -file ".\Testscripts\Linux\utils.sh"
```
Comment (lizzha) [inline Libraries/CommonFunctions.psm1]: please use copy-remotefiles, which wraps upload-remotefile, and it will do the dos2unix operation

**Pull request merged**